src/main/java/com/example/pet_shelter/controller/AnimalRESTController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.pet_shelter.controller;
+
+
+import com.example.pet_shelter.entity.Animal;
+import com.example.pet_shelter.service.AnimalService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
it is not a mistake, but you can make @RequestMapping("/api/animals"), after all It is animal controller